appengine/go11x/tasks/handle_task/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Use the offical Golang image to create a build artifact.
 # This is based on Debian and sets the GOPATH to /go.
 # https://hub.docker.com/_/golang
-FROM golang:1.13 as builder
+FROM golang:1.14 as builder
```

Comment:
docker build and run works, but GAE is only in beta for 1.14. Not sure if this should block or if it's a "feature" that the Dockerfile can be a forward version. Also not clear why an App Engine sample on using Tasks has a Dockerfile... should this be in a tasks/ directory?